featuretools/variable_types/variable.py

```diff
@@ -1,7 +1,14 @@
 import numpy as np
 import pandas as pd
 
-from featuretools.utils.gen_utils import find_descendents
+from featuretools.utils.gen_utils import camel_to_snake, find_descendents
+
+
+class ClassNameDescriptor(object):
+    """Descriptor to derive the a variable's type_string from it's class name
```

Comment:
I think the description could be a little more general, we might use this descriptor for other classes besides `Variable` and it's subclasses